cost/scheduled_reports_with_estimates/README.md

```diff
@@ -0,0 +1,44 @@
+Scheduled Report
```

Comment:
Looks like this readme was copied from the schedule report.  Please update to match the policy, and is needs to be in Markdown format.  See the README_GUIDELINES

Reply:
done